Optimize shapes following editorial review

justinwb opened the pull request:
Resolves #151

ericprud reviewed (changes requested):
I PR'd your PR to add three PREFIX decls.
tested in https://shexspec.github.io/ace-shexc-user/

justinwb commented:
Removed the ref file for now per @elf-pavlik's request. PR from @ericprud is merged in. I removed the data grant scopes island - was a remnant of how things were structured before this PR optimized.
